simcon_project/simcon_project/admin.py

```diff
@@ -1,9 +0,0 @@
-from django.contrib import admin
```

Comment:
keep this until we make an admin page

Reply:
Same reply as to Michael's comment above.